.pre-commit-config.yaml

```
@@ -1,11 +1,11 @@
 ---
 repos:
   - repo: https://github.com/UCL-MIRSG/.github
-    rev: v0.43.0
+    rev: v0.48.0
     hooks:
       - id: mirsg-hooks
   - repo: https://github.com/renovatebot/pre-commit-hooks
-    rev: 37.44.0
+    rev: 37.233.1
```

I've intentionally [forbidden updates to non-major updates](https://github.com/UCL-MIRSG/.github/blob/3c0c3f3d63ab5c1fa7e1b5f836bf5aaab818cdc5/renovate/default-config.json#L40-L43) because the downloading of this hook is incredibly slow. @renovatebot [update this hook multiple times a day](https://github.com/renovatebot/pre-commit-hooks/tags)!
```suggestion
    rev: 37.44.0
```

---

ah, thanks for spotting that!